lib/matplotlib/streamplot.py

```diff
@@ -97,6 +118,18 @@ def streamplot(axes, x, y, u, v, density=1, linewidth=None, color=None,
     mask = StreamMask(density)
     dmap = DomainMap(grid, mask)
 
+    if integration_max_step_scale <= 0.0:
+        raise ValueError(
+            "The value of integration_max_step_scale must be > 0, " +
+            f"got {integration_max_step_scale=}"
```

Comment:
There's no need for `=` here; the parameter is already named in the beginning of the sentence.
```suggestion
            f"got {integration_max_step_scale}"
```